README.md

```diff
@@ -1,24 +1,37 @@
-# Crud With Validations Lab
+# CRUD With Validations Lab
 
-## Objectives
+Your goal in this lab is to create a thorough CRUD interface for one model, the
```

Comment:
I'd like to see more of a story here. Let's think of a way to describe this as a feature the student is building, not just a list of tasks. Maybe they're building a basic system to catalog their music or something for a record exec to keep track of their artists' releases. Something to jazz it up a bit.